### ISSUE TITLE
IP address validation only validates lists, not individual entries

### ISSUE THREAD

**Issue by jjguy** [carbonblack/cb-taxii-connector#14]
https://github.com/carbonblack/cb-taxii-connector/blob/master/cbopensource/connectors/taxii/cybox_parse.py#L169

The parsing for IPv4 addresses validates the format of lists of IPs via `validate_ip_address()` and fails cleanly on unexpected inputs, but does not validate individual entries.  (e.g., the else block)

If an individual Address entry is malformed, it results in the bad IP address being included in the list of IOCs and a hard fail later in processing (e.g., on `feed.dump()`). 

We should update the else block to call `validate_ip_address()`

-----
Below is stack trace that triggered this, note the IP address we tried to parse in the feed validation contains an URL.  That's a bug in the feed - our bug is we failed hard instead of cleanly.

```
2017-05-17 08:51:11,904 Sending Poll_Request to https://analysis.foo.com/taxii-data
2017-05-17 08:51:18,787 content blocks read: 2194
2017-05-17 08:51:18,788 current number of reports: 4701
2017-05-17 08:51:18,788 Found 4701 new reports.
2017-05-17 08:51:18,804 Total number of reports: 8860
2017-05-17 08:51:19,363 Traceback (most recent call last):
  File "cbopensource/connectors/taxii/bridge.py", line 468, in runner
  File "cbopensource/connectors/taxii/bridge.py", line 446, in perform
  File "cbopensource/connectors/taxii/bridge.py", line 344, in _import_collection
  File "cbopensource/connectors/taxii/cb_feed_util.py", line 116, in build_feed_data
  File "cbfeeds/feed.py", line 28, in dump
  File "cbfeeds/feed.py", line 91, in validate
  File "cbfeeds/feed.py", line 348, in validate
CbInvalidReport: Malformed IPv4 (1.2.3.4/zeco/panelnew/gate.php) addr in IOC list for report foo-Observable-eeb1d830-ac39-49f0-9317-57ae9b42eb3b
```

**Comment by askthedragon** [comment]
Thanks for reporting this.  Should have a new release out in the next couple of days

**Comment by askthedragon** [comment]
see commit 6356774e4b8be54f6e9d3aeeb7bc08f51a02bd91

**Comment by askthedragon** [comment]
Fixed with 6356774
